FIP0036/post_process/counting.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Sun Sep 18 23:27:17 2022
+
+@author: juan
+"""
+
+import json
+import datautils as utils
+
+def countVote(vote,groups_of_voters,datasets,signatures):
+
+    list_of_deals=datasets['deals']
+    list_of_miners=datasets['miners']
+    list_of_addresses_and_ids=datasets['addresses']
+    list_of_core_devs=datasets['core']
+    list_of_powers=datasets['powers']
+    
+    signature=json.loads(vote["signature"])
+    X = signature["signer"]
+    signature=utils.addShortAndLongId(signature=signature,
+                list_of_addresses_and_ids=list_of_addresses_and_ids)
+    #
+    # checks if it;s a core dev and adds headcount
+    #
+    is_core_dev =utils.is_in_list(X, list_of_core_devs)
+    
+    
+    
+    if is_core_dev:
+        groups_of_voters["core"].validateAndAddVote(signature)
+    
+    #
+    # checks if exists and adds balance
+    #
+    #exists = utils.is_in_list(X, list_of_addresses)
+    #if exists:
+    groups_of_voters["token"].validateAndAddVote(signature)
```

Comment:
Can we confirm that we are conforming to [this comment](https://github.com/filecoin-project/FIPs/discussions/464#discussioncomment-3656681) from jennijuju? There it is stated that: 
> the balance of the miner actor's available balance will also be added to the Token Holders count

The `validateAndAddVote` method does the following for group 4: `quantity=int(signature['balance'])`. Does the `balance` field include the available balance of the miner actor?